### PR TITLE
Add info about logging into shell before removing /mnt/data

### PR DESCRIPTION
**Pull request by vadasambar**
details
----------
- rm -rf should be run after logging into the node shell

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ kubectl delete pvc task-pv-claim
 kubectl delete pv task-pv-volume
 ```
 
-Remove the file:
+Open the shell to the Node (if you are using minikube, do `minikube ssh`) and remove the file:
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md]: How about reusing the text from the start of the task:
>  Open a shell to the Node in your cluster. How you open a shell depends on how
you set up your cluster. For example, if you are using Minikube, you can open a
shell to your Node by entering `minikube ssh`.

?

Comment (vadasambar) [reply content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md]: >Open the shell to the Node in your cluster again. How you open a shell depends on how
you set up your cluster. For example, if you are using Minikube, you can open a
shell to your Node by entering minikube ssh.
I made a minor change to this. How does this look?

Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md]: @vadasambar sure, feel free to go ahead and tweak the commit in this pull request to match.

Comment (vadasambar) [reply content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md]: I have made the change. Please let me know if it looks good.

**Pull request merged**